include/exadg/operators/interior_penalty_parameter.h

```diff
@@ -89,16 +89,21 @@ calculate_penalty_parameter(
 }
 
 /*
- *  This function returns the penalty factor of the interior penalty method
- *  for quadrilateral/hexahedral elements for a given polynomial degree of
- *  the shape functions and a specified penalty factor (scaling factor).
+ *  This function returns the penalty factor of the interior penalty method for
+ *  quadrilateral/hexahedral or for triangular/tetrahedral elements for a given
+ *  polynomial degree of the shape functions, a specified penalty factor
+ *  (scaling factor) and the dimension of the problem.
  */
 
-template<typename Number>
+template<int dim, typename Number>
 Number
-get_penalty_factor(unsigned int const degree, Number const factor = 1.0)
+get_penalty_factor(unsigned int const degree, bool const use_simplex, Number const factor = 1.0)
 {
-  return factor * (degree + 1.0) * (degree + 1.0);
+  // use penalty factor for simplex elements according to Shahbazi (2005)
+  if(use_simplex)
+    return factor * (degree + 1.0) * (degree + dim) / dim;
+  else
+    return factor * (degree + 1.0) * (degree + 1.0);
```

Comment:
We then also have to introduce a reference here for quad/hex for completeness. You can find the original reference in my PhD thesis. Maybe it is good to also provide the Journal, Issue number, etc. as you did in the description of this PR.

Reply:
addressed in afa84cdcbb31076a780301311fece2f03099716e.